packages/ui/components/dialog/test/lion-dialog.test.js

```diff
@@ -72,4 +85,117 @@ describe('lion-dialog', () => {
       expect(nestedDialogEl.opened).to.be.true;
     });
   });
+
+  describe('focus handling', () => {
```

Comment:
Since lion-dialog is only a "convenience component" with a preconfigured OverlayController, can we move these tests to OverlayController?